index.html

```diff
@@ -1,22 +1,95 @@
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <meta name="description" content="This is a description of my blog">
+  <title> MY Javascript Blog</title>
+  <link href="//fonts.googleapis.com/css?family=Roboto:400,500,300" rel="stylesheet" type="text/css" />
+  <link rel="stylesheet" href="css/style.css" />
+  <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+
+</head>
+
+<body>
+  <!-- Add your HTML markup here -->
+
+  <header>
+    <h1> JS Documentation</h1>
+    <nav>
+      <ul>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Blog</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section class="article">
+      <article>
+        <header>
+          <h2>
+            What is Lorem Ipsum?
+          </h2>
+          <img
+            src="https://dvg5hr78c8hf1.cloudfront.net/2016/06/17/13/45/01/cc2fac88-2f81-4cc5-9ec9-334042781fd0/1*OsjnQFK1i6CkjXQmTErAtw.jpeg"
+            alt=" JavaScript image">
+          <p>
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+            industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+            scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into
+            electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of
+            Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like
+            Aldus PageMaker including versions of Lorem Ipsum.
+          </p>
+      </article>
+      </header>
+
+      <article>
+        <header>
+          <h2>
+            Why do we use it?
+          </h2>
+          <img src="https://www.tutorialrepublic.com/lib/images/javascript-illustration.png" alt=" JavaScript image">
+          <p>
+            It is a long established fact that a reader will be distracted by the readable content of a page when
+            looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of
+            letters, as opposed to using 'Content here, content here', making it look like readable English. Many
+            desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a
+            search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved
+            over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
+          </p>
+      </article>
```

Comment:
Well done you have added a title using the `<h2>` tag. 👍🏽
Please see below for improvements:
1. You are missing a closing tag `</header>` in between the `<articles></articles>` block. Actually, you have instead put the closing `</header>` tag outside the `<articles></articles>` block
2. The task also requires you to include a `<summary></summary>` block within the `<articles></articles>` block. Please can you add that to your code.